src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -457,7 +457,18 @@ private void drawListFG(final int offsetX, final int offsetY, final int mouseX,
                 if (pendingStack != null && pendingStack.getStackSize() > 0) {
                     String str = GuiText.ToCraft.getLocal() + ": "
                             + ReadableNumberConverter.INSTANCE.toWideReadableForm(pendingStack.getStackSize());
-                    final int w = 4 + this.fontRendererObj.getStringWidth(str);
+                    int w = 4 + this.fontRendererObj.getStringWidth(str);
+                    this.fontRendererObj.drawString(
+                            str,
+                            (int) ((x * (1 + sectionLength) + xo + sectionLength - 19 - (w * 0.5)) * 2),
+                            (y * offY + yo + 6 - negY + downY) * 2,
+                            GuiColors.CraftConfirmToCraft.getColor());
+
+                    downY += 5;
+                    str = GuiText.ToCraftRequests.getLocal() + ": "
```

Comment:
I think It also can be helpful on crating status screen, but there are one problem: on that screen it can be on 4th line and will be unreadable. May be we can find other way to show this information? 
I've two thoughts here
- utilize short format and write `To Craft: 9200k ◊: 9.2M`
- utilize color gradation from blue to red for pattern crafts, using all nodes in recipe

Reply:
I don't think that this info is needed on the crafting status. It is here to know which patterns are not multiplied to optimize them.
We can always add it later.
Also there is currently a way that 4 lines can appear on crafting plan:
![image](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/assets/53441451/134b911a-71db-4fa1-b3a2-d481d83fce8e)
I dont think it looks that bad